catalog/solr/catalog-solr-offline-gazetteer/src/main/java/ddf/catalog/solr/offlinegazetteer/GazetteerQueryOfflineSolr.java

```diff
@@ -0,0 +1,465 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package ddf.catalog.solr.offlinegazetteer;
+
+import static ddf.catalog.solr.offlinegazetteer.GazetteerConstants.NAMES;
+import static ddf.catalog.solr.offlinegazetteer.GazetteerConstants.SUGGEST_DICT;
+import static ddf.catalog.solr.offlinegazetteer.GazetteerConstants.SUGGEST_DICT_VALUE;
+import static ddf.catalog.solr.offlinegazetteer.GazetteerConstants.SUGGEST_Q;
+
+import com.google.common.collect.ImmutableMap;
+import ddf.catalog.data.types.Core;
```

Comment:
✏️ The fact that these field names match the catalog metacard attributes is now an incidental implementation detail of how the default suggestor is configured in the schema.  I would recommend removing the use of these attributes since these are not actually metacard attributes.